### ISSUE TITLE
Keep serde skipped field in API Spec

### ISSUE THREAD

**Issue by VirusBLITZ** [juhaku/utoipa#868]
Say I have a struct like
```rust
#[derive(Debug, Serialize, Deserialize, Clone, ToSchema)]
pub struct Client {
    #[serde(skip_deserializing)]
    pub test: Option<bool>,
}
```
the field `test` is being ignored because of the serde `skip_deserializing` macro, is there a way to keep it in the spec?

**Comment by hutchisr** [comment]
I agree it would be very nice to have some way to not skip these fields under some circumstances. One workaround is to create another struct with the unskipped field and the original struct as properties and use `#[serde(flatten)]`. From a project I'm working on (also doing the same with sqlx in this case):

```rust
#[derive(FromRow, Serialize, Deserialize, ToSchema, Debug)]
#[serde(rename_all = "camelCase")]
pub struct Tag {
    #[serde(skip_deserializing)]
    pub id: Uuid,
    // ...rest of struct fields
}

#[derive(FromRow, Serialize, Deserialize, ToSchema, Debug)]
#[serde(rename_all = "camelCase")]
pub struct TagWithId {
    pub id: Uuid,
    #[sqlx(flatten)]
    #[serde(flatten)]
    pub tag: Tag,
}
```

**Comment by VirusBLITZ** [comment]
> I agree it would be very nice to have some way to not skip these fields under some circumstances. One workaround is to create another struct with the unskipped field and the original struct as properties and use `#[serde(flatten)]`. From a project I'm working on (also doing the same with sqlx in this case):
> 
> ```rust
> #[derive(FromRow, Serialize, Deserialize, ToSchema, Debug)]
> #[serde(rename_all = "camelCase")]
> pub struct Tag {
>     #[serde(skip_deserializing)]
>     pub id: Uuid,
>     // ...rest of struct fields
> }
> 
> #[derive(FromRow, Serialize, Deserialize, ToSchema, Debug)]
> #[serde(rename_all = "camelCase")]
> pub struct TagWithId {
>     pub id: Uuid,
>     #[sqlx(flatten)]
>     #[serde(flatten)]
>     pub tag: Tag,
> }
> ```

Thanks for the reply, seems like a solution that I can live with 😄

**Comment by Drifter-J** [comment]
Resolved by workaround.
